unittest/contact-dynamics-derivatives.cpp

```diff
@@ -27,16 +27,6 @@ BOOST_AUTO_TEST_SUITE(BOOST_TEST_MODULE)
 using namespace Eigen;
 using namespace pinocchio;
 
-PINOCCHIO_STD_VECTOR_WITH_EIGEN_ALLOCATOR(pinocchio::RigidConstraintData)
```

Comment:
Why such a choice?

Reply:
Just a gambit. Contact dynamics derivatives is always the only one failing, so I'm trying to see why that is.